### ISSUE TITLE
Broken images with hardware acceleration

### ISSUE THREAD

**Issue by bertho-zero** [brave/brave-browser#27449]
## Description 

I often encounter images that are broken while browsing, it displays correctly if I change tabs before coming back to it.

## Steps to Reproduce

   1. Go to a search engine for images
   2. Search 'cat'
   3. Zoom out fully

## Actual result:
![image](https://user-images.githubusercontent.com/8525267/208455395-623a438d-38a6-4bbc-987d-06e749a59509.png)

## Expected result:
![image](https://user-images.githubusercontent.com/8525267/208455618-e8fb15f8-2d4c-4e65-bbc6-5ca8b63869da.png)

## Reproduces how often: 
I regularly have broken images while browsing normally.

## Brave version (brave://version info)

Brave: 1.46.144 Chromium: 108.0.5359.128 (Build officiel) (64 bits)
Révision: 1cd27afdb8e5d057070c0961e04c490d2aca1aa0-refs/branch-heads/5359@{#1185}
Système d'exploitation: Linux

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

## Miscellaneous Information:
By disabling hardware acceleration I no longer have this problem.

**Comment by rebron** [comment]
cc: @wknapik Can you reproduce this issue too?

**Comment by wknapik** [comment]
@rebron yes, I reported this as part of https://github.com/brave/brave-browser/issues/27417

**Comment by wknapik** [comment]
Still an issue as of 1.47.171

**Comment by CRCinAU** [comment]
I also get this viewing the [Proxmox](https://proxmox.com) UI. Anything that shows a graph will do this:

![image](https://github.com/brave/brave-browser/assets/3992884/eac1d48d-b0e7-4c15-9de3-eab1992550c7)
